scripts/remove/znc.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 #ZNC Removal
 
-systemctl disable znc >> /dev/null 2>&1
-systemctl stop znc >> /dev/null 2>&1
+systemctl disable -q znc >> /dev/null 2>&1
+systemctl stop -q znc >> /dev/null 2>&1
```

Comment:
pls no `/dev/null` no more

Reply:
This can be changed. Historically, remove scripts have never really imported log functions which is why in general remove scripts are mostly outputting to dev null.

Reply:
I removed all /dev/null references in this package, I'll make a raid on that in some other PR